source/hint_list.cpp

```diff
@@ -2067,6 +2081,51 @@ void HintTable_Init() {
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_2] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling twice# grants",               /*french*/"#parier# donne [2]",                     /*spanish*/"#los juegos de azar# revelan [2]"},
+                       Text{"the #third or fourth chest# contains",               /*french*/"#parier# donne [2]",                     /*spanish*/"#los juegos de azar# revelan [2]"},
+                     }, {},
+                       //clear text
+                       Text{"the #second locked room# in the chest game contains", /*french*/"la #chasse aux trésors# donne [2]", /*spanish*/"#el Cofre del Tesoro# premia [2]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_3] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 3 times# grants",               /*french*/"#parier# donne [3]",                     /*spanish*/"#los juegos de azar# revelan [3]"},
+                       Text{"the #fifth or sixth chest# contains",               /*french*/"#parier# donne [3]",                     /*spanish*/"#los juegos de azar# revelan [3]"},
+                     }, {},
+                       //clear text
+                       Text{"the #third locked room# in the chest game contains", /*french*/"la #chasse aux trésors# donne [3]", /*spanish*/"#el Cofre del Tesoro# premia [3]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_4] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 4 times# grants",               /*french*/"#parier# donne [4]",                     /*spanish*/"#los juegos de azar# revelan [4]"},
+                       Text{"the #seventh or eighth chest# contains",               /*french*/"#parier# donne",                     /*spanish*/"#los juegos de azar# revelan [4]"},
+                     }, {},
+                       //clear text
+                       Text{"the #fourth locked room# in the chest game contains", /*french*/"la #chasse aux trésors# donne [4]", /*spanish*/"#el Cofre del Tesoro# premia [4]"}
+  );
+
+    hintTable[MARKET_TREASURE_CHEST_GAME_ITEM_5] = HintText::Sometimes({
+                       //obscure text
+                       Text{"#gambling 5 times# grants",               /*french*/"#parier# donne [5]",                     /*spanish*/"#los juegos de azar# revelan [5]"},
+                       Text{"the #ninth or tenth chest# contains",               /*french*/"#parier# donne [5]",                     /*spanish*/"#los juegos de azar# revelan [5]"},
+                     }, {},
+                       //clear text
+                       Text{"the #fourth locked room# in the chest game contains", /*french*/"la #chasse aux trésors# donne", /*spanish*/"#el Cofre del Tesoro# premia"}
```

Comment:
#parier cinq fois# donne
le #neuvième ou dixième coffre à jeu# contient

la #cinquième salle# de la chasse aux trésors contient

change "fourth" to "fifth" in the clear hint